app/controllers/AFTController.scala

```diff
@@ -242,15 +242,14 @@ class AFTController @Inject()(
       }
   }
 
-  private def detailsJsLogic(js:JsValue,version:AFTVersion): VersionsWithSubmitter = {
+
+
+  private def detailsJsLogic(js: JsValue, version: AFTVersion): VersionsWithSubmitter = {
+    import models.AFTSubmitterDetails._
     logger.warn(s"detailsJsLogic started for: ${version.reportVersion}")
-    val transform = js.transform(aftDetailsTransformer.transformToUserAnswers) match {
```

Comment:
We were reading all the data to userAnswers first, then when we get user answers we read it again to get AFTSubmitterDetails. 

I have changed it to read straight into AFTSubmitterDetails.